ui/src/app/stores/workflows/workflows.effects.spec.ts

```diff
@@ -75,7 +75,7 @@ describe('WorkflowsEffects', () => {
       const getProjectsResponse = cold('-a|', { a: projects });
       const expected = cold('--a', { a: {
           type: WorkflowsActions.INITIALIZE_WORKFLOWS_SUCCESS,
-          payload: {projects: projects, workflows: [].concat(projects.map((project) => project.workflows))}
+          payload: {projects: projects, workflows: [].concat(...projects.map((project) => project.workflows))}
```

Comment:
Why do you need to `...` here?

Reply:
It splits array of arrays into this concat(...items: ConcatArray<T>[])